service/http.balancer/src/test/java/org/kaazing/gateway/service/http/balancer/WsxBalancerIT.java

```diff
@@ -54,7 +53,6 @@
                             .allowOrigin("*")
```

Comment:
We need at least 4 tests.  WSX with kl=y (sends balancer bytes), WSX with out kl=y ( sends wrapped 302),  WS with kl=y (sends balancer bytes), WS with out kl=y ( sends 302).  I believe we only have 2 cases covered in the original tests